website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: The Anatomy of Unleash
+---
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+Unleash has many different components on different levels. This guide is here to give you a conceptual overview over how Unleash works. It covers the various components that exist within an Unleash system and how they interact with each other and with external applications. The diagrams are intended to help you understand the fundamental building blocks, such as [projects](../user_guide/projects.md), [environments](../user_guide/environments.md), [variants](../advanced/feature-toggle-variants.md) and, of course, [feature toggles](../reference/feature-toggles.mdx).
```

Comment:
Personal feeling but I think the first sentence here is redundant

Reply:
Agreed, let's remove it.